core/server/proxy/src/main/java/alluxio/proxy/s3/S3RestServiceHandler.java

```diff
@@ -701,40 +757,6 @@ private void checkUploadId(AlluxioURI multipartTemporaryDir, long uploadId) thro
     }
   }
 
-  private List<URIStatus> listObjects(AlluxioURI uri, ListBucketOptions listBucketOptions)
```

Comment:
why is this removed?

Reply:
It's an unused code that became unused after previous commit when list Bucket was fixed.